book/_toc.yml

```diff
@@ -1,26 +1,42 @@
 # Table of content
 # Learn more at https://jupyterbook.org/customize/toc.html
-#
 - file: intro
-- file: application
-- file: schedule
-- file: team
-- file: norms/index
-  sections:
+- part: Details
+  chapters:
+  - file: application
+  - file: schedule
+  - file: team
+  - file: norms/index
+    sections:
     - file: norms/CoC
     - file: norms/community
-- file: projects/index
-  sections:
-    - file: projects/project_initialization
-    - file: projects/project_roles
-    - file: projects/example_workflow
-- file: preliminary/index
-  sections:
-    - file: preliminary/conda
-    - file: preliminary/github
-    - file: preliminary/jupyterhub
-    - file: preliminary/jupyterhub_fork_clone
-    - file: preliminary/earthdata
-- file: tutorials/index
-  sections:
-    - file: tutorials/raster/intro-ipyleaflet
```

Comment:
let's keep this one in for now, it's nice to have an example rendered notebook - https://uwhackweek.github.io/jupyterbook-template/tutorials/raster/intro-ipyleaflet.html